[FEA] Make DataFrame.clean_renderable_dataframe() and DataFrame.get_renderable_dataframe non-public methods

Currently, the helper methods `DataFrame.clean_renderable_dataframe` and `DataFrame.get_renderable_dataframe` appear as part of the public API. They should be converted to underscore methods `_clean_renderable_dataframe` and `_get_renderable_dataframe` instead.

Preface: This comment is mostly unrelated to the issue. I am not sure if there is a better place to ask this question.

I have made the changes but would like to test locally. I usually use WSL as my dev environment which does not support CUDA. In an attempt to work around this (using windows), I attempted to build the application using Docker. However, I am getting the following build errors:
```
[ 95%] Building CXX object src/arrow/CMakeFiles/arrow_objlib.dir/ipc/reader.cc.o
[ 96%] Building CXX object src/arrow/CMakeFiles/arrow_objlib.dir/ipc/json_simple.cc.o
g++-5: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-5/README.Bugs> for instructions.
src/arrow/dataset/CMakeFiles/arrow_dataset_objlib.dir/build.make:95: recipe for target 'src/arrow/dataset/CMakeFiles/arrow_dataset_objlib.dir/file_base.cc.o' failed
make[5]: *** [src/arrow/dataset/CMakeFiles/arrow_dataset_objlib.dir/file_base.cc.o] Error 4
make[5]: *** Waiting for unfinished jobs....
g++-5: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-5/README.Bugs> for instructions.
```

My assumptions are:
1) That I do not need `nvidia-docker2` anymore [Based on the Nvidia Docker repository]( https://github.com/NVIDIA/nvidia-docker). "Note that with the release of Docker 19.03, usage of nvidia-docker2 packages are deprecated since NVIDIA GPUs are now natively supported as devices in the Docker runtime."
2) That the build process should run within the container.

I was wondering if this is:
a. A general docker build issue, or
b. a circumstance of my environment, or
c. an actual bug?

Finally, do you think using the docker image would be a viable method to test my changes or should I attempt to use CMD/PowerShell?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -36,6 +36,7 @@
 
 ## Improvements
 
+- PR #5235 Make DataFrame.clean_renderable_dataframe() and DataFrame.get_renderable_dataframe non-public methods
 - PR #4995 Add CMake option for per-thread default stream
 - PR #5033 Fix Numba deprecations warnings with Numba 0.49+
 - PR #4950 Fix import errors with Numba 0.49+

--- a/python/cudf/cudf/core/dataframe.py
+++ b/python/cudf/cudf/core/dataframe.py
@@ -901,7 +901,7 @@ class DataFrame(Frame, Serializable):
         ncols = 19 if ncols in [20, 21] else ncols
         return ncols, nrows
 
-    def clean_renderable_dataframe(self, output):
+    def _clean_renderable_dataframe(self, output):
         """
         the below is permissible: null in a datetime to_pandas() becomes
         NaT, which is then replaced with null in this processing step.
@@ -920,7 +920,7 @@ class DataFrame(Frame, Serializable):
             )
         return "\n".join(lines)
 
-    def get_renderable_dataframe(self):
+    def _get_renderable_dataframe(self):
         """
         takes rows and columns from pandas settings or estimation from size.
         pulls quadrents based off of some known parameters then style for
@@ -984,12 +984,12 @@ class DataFrame(Frame, Serializable):
         return output
 
     def __repr__(self):
-        output = self.get_renderable_dataframe()
-        return self.clean_renderable_dataframe(output)
+        output = self._get_renderable_dataframe()
+        return self._clean_renderable_dataframe(output)
 
     def _repr_html_(self):
         lines = (
-            self.get_renderable_dataframe()
+            self._get_renderable_dataframe()
             .to_pandas()
             ._repr_html_()
             .split("\n")
@@ -1004,7 +1004,7 @@ class DataFrame(Frame, Serializable):
         return "\n".join(lines)
 
     def _repr_latex_(self):
-        return self.get_renderable_dataframe().to_pandas()._repr_latex_()
+        return self._get_renderable_dataframe().to_pandas()._repr_latex_()
 
     # unary, binary, rbinary, orderedcompare, unorderedcompare
     def _apply_op(self, fn, other=None, fill_value=None):

